docs/setuptools.txt

```diff
@@ -1223,121 +1223,53 @@ the quoted part.
 Distributing a ``setuptools``-based project
 ===========================================
 
-Using ``setuptools``...  Without bundling it!
----------------------------------------------
+Detailed instructions to distribute a setuptools project can be found at 
+`Packaging project tutorials`_.
 
-.. warning:: **ez_setup** is deprecated in favor of PIP with **PEP-518** support.
+.. _Packaging project tutorials: https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives
 
-Your users might not have ``setuptools`` installed on their machines, or even
-if they do, it might not be the right version.  Fixing this is easy; just
-download `ez_setup.py`_, and put it in the same directory as your ``setup.py``
-script.  (Be sure to add it to your revision control system, too.)  Then add
-these two lines to the very top of your setup script, before the script imports
-anything from setuptools:
+Before you begin. make sure you have the latest versions of setuptools and wheel::
 
-.. code-block:: python
+    python3 -m pip install --user --upgrade setuptools wheel
 
-    import ez_setup
-    ez_setup.use_setuptools()
+To build a setuptools project, run this command from the same directory where
+setup.py is located::
 
-That's it.  The ``ez_setup`` module will automatically download a matching
-version of ``setuptools`` from PyPI, if it isn't present on the target system.
-Whenever you install an updated version of setuptools, you should also update
-your projects' ``ez_setup.py`` files, so that a matching version gets installed
-on the target machine(s).
+    python3 setup.py sdist bdist_wheel
```

Comment:
I'm inclined to leave this for a subsequent PR, but I think it would be preferable not to suggest `python setup.py bdist_wheel` now that `pip wheel` is a thing.

Reply:
Would it be better if we create a new issue on both setuptools and [python-packaing-user-guide](https://github.com/pypa/python-packaging-user-guide/blob/master/source/tutorials/packaging-projects.rst#generating-distribution-archives) because I think both places will need changes.
There is also [Issue #539](https://github.com/pypa/python-packaging-user-guide/issues/539) with some existing discussion regarding the suggestion on uploading sdists.

Reply:
We'll just defer this for now. `pip wheel` doesn't do exactly what `python setup.py bdist_wheel` does, so it may be a situation where we solve #1515 and then migrate the guide over to `setuptools <x>` instead.